### PR TITLE
Use meta noindex tag instead of robots.txt

### LINKED ISSUE #192: Use meta noindex tag
Use meta noindex tag instead of robots.txt

https://support.google.com/webmasters/answer/93710?hl=en

### PR DESCRIPTION
**Pull request by jasonharrison**
Use meta noindex tag instead of robots.txt

https://support.google.com/webmasters/answer/93710?hl=en

If running on freeboardgame.org, return 404 for /robots.txt, else, return a restrictive robots.txt

Closes #192

### CODE REVIEW

File: src/server.tsx [hunk 1]
```diff
@@ -27,16 +27,28 @@ const DEV = !PROD;
...
-  result = result.replace('<title>FreeBoardGame.org</title>', `<title>${title}</title>`);
-  result = result.replace(
-    '<meta name="Description" content="">',
-    `<meta name="Description" content="${description}">`);
+
+  result = result.replace('<title>FreeBoardGame.org</title>', `<title>${metadata.title}</title>`);
+
+  if (metadata.description) {
+    result = result.replace(
+      '    <meta name="description" content="">\n',
```
Comment (flamecoals) [inline src/server.tsx]: can we remove the spaces inside the string being replaced? I think this is a very easy source of failure.

File: src/server.tsx [hunk 2]
```diff
@@ -79,9 +91,7 @@ const startServer = async () => {
   server.app.use(router.allowedMethods());
   server.app.use(async (ctx: any, next: any) => {
```
Comment (flamecoals) [inline src/server.tsx]: This block can be removed altogether, no? We dont rely on robots.txt anymore?

File: src/server.tsx [hunk 3]
```diff
@@ -79,9 +91,7 @@ const startServer = async () => {
...
     if (ctx.request.path === '/robots.txt') {
-      if (ctx.request.hostname.toLowerCase() === 'freeboardgame.org') {
-        ctx.response.body = OPEN_ROBOTS_TXT;
-      } else {
+      if (ctx.request.hostname.toLowerCase() !== 'freeboardgame.org') {
         ctx.response.body = RESTRICTIVE_ROBOTS_TXT;
```
Comment (flamecoals) [inline src/server.tsx]: We can probably remove the RESTRICTIVE and OPEN_ROBOTS_TXT constant too

File: src/server.tsx (hunk 1 above)
Comment (jasonharrison) [reply src/server.tsx]: Removed, it was so the HTML was aligned

File: src/server.tsx (hunk 3 above)
Comment (jasonharrison) [reply src/server.tsx]: OPEN_ROBOTS_TXT is not present.

RESTRICTIVE_ROBOTS_TXT is served when the Host header is not freeboardgame.org, which I thought was desired behavior.

**Pull request merged**